platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/capabilities/CapabilitiesUseCasesIntegrationTest.groovy

```diff
@@ -410,7 +410,7 @@ class CapabilitiesUseCasesIntegrationTest extends AbstractModuleDependencyResolv
 
         then:
         if (failOnVersionConflict) {
-            failure.assertHasCause("Conflict found for the following module:\n  - org.ow2.asm:asm latest version of capability asm:asm")
+            failure.assertHasCause("Conflict found for module 'org.ow2.asm:asm': latest version of capability asm:asm")
```

Comment:
The colon is strange here too.  Can this sometimes show multiple lines?

Here are some ideas like above:
```
            failure.assertHasCause("Conflict found for module 'org.ow2.asm:asm' on latest version of capability asm:asm")
            failure.assertHasCause("Conflict found on latest version of capability asm:asm for module 'org.ow2.asm:asm'.")
```